src/main/kotlin/no/nav/syfo/kartleggingssporsmal/infrastructure/kafka/KafkaOppfolgingstilfellePersonDTO.kt

```diff
@@ -1,52 +1,14 @@
 package no.nav.syfo.kartleggingssporsmal.infrastructure.kafka
 
-import no.nav.syfo.kartleggingssporsmal.domain.Oppfolgingstilfelle
-import no.nav.syfo.shared.domain.Personident
-import no.nav.syfo.shared.util.tomorrow
+import no.nav.syfo.kartleggingssporsmal.infrastructure.clients.oppfolgingstilfelle.OppfolgingstilfelleDTO
 import java.time.LocalDate
 import java.time.OffsetDateTime
-import java.util.*
 
 data class KafkaOppfolgingstilfellePersonDTO(
     val uuid: String,
     val createdAt: OffsetDateTime,
     val personIdentNumber: String,
-    val oppfolgingstilfelleList: List<KafkaOppfolgingstilfelle>,
+    val oppfolgingstilfelleList: List<OppfolgingstilfelleDTO>,
     val referanseTilfelleBitUuid: String,
-    val referanseTilfelleBitInntruffet: OffsetDateTime,
     val dodsdato: LocalDate?,
 )
-
-data class KafkaOppfolgingstilfelle(
-    val gradertAtTilfelleEnd: Boolean,
-    val arbeidstakerAtTilfelleEnd: Boolean,
-    val start: LocalDate,
-    val end: LocalDate,
-    val antallSykedager: Int?,
-    val virksomhetsnummerList: List<String>,
-)
-
-fun KafkaOppfolgingstilfellePersonDTO.toLatestOppfolgingstilfelle(): Oppfolgingstilfelle? =
-    this.oppfolgingstilfelleList
-        .filter { isNotTilfelleInTheFuture(it) }
-        .maxByOrNull { it.start }
-        ?.let { this.toOppfolgingstilfelle(it) }
```

Comment:
Hvis vi hadde holdt ting separat fra hverandre, så ville denne blitt duplisert to steder. Nå lever den heller i den felles `Oppfolgingstilfelle`-klassen